config/jobs-chromeos.yaml

```diff
@@ -551,3 +551,19 @@ jobs:
   tast-ui-arm64-qualcomm: *tast-ui-job
   tast-ui-x86-intel: *tast-ui-job
   tast-ui-x86-amd: *tast-ui-job
+
+  v4l2-decoder-conformance-av1:
```

Comment:
Given that we'll have other similar test suites where the only difference will be the `testsuite` and `decoders`  parameters, I suggest defining an anchor in this file as a base to include/extend right away - something like:
```
  v4l2-decoder-conformance-job: &v4l2-decoder-conformance-job
    template: 'v4l2-decoder-conformance-template.jinja2'
    kind: test
    params: &v4l2-decoder-conformance-job-params
      nfsroot: 'https://storage.staging.kernelci.org/rootfs/bookworm-gst-fluster/20240430.0/{debarch}/'
      job_timeout: 30
      videodec_parallel_jobs: '1'
      videodec_timeout: 90
    rules:
      tree:
        - mainline
        - next
```

The AV1 test job definition can then be simplified to:

```
  v4l2-decoder-conformance-av1:
    <<: *v4l2-decoder-conformance-job
    params:
      <<: *v4l2-decoder-conformance-job-params
      testsuite: 'AV1-TEST-VECTORS'
      decoders:
        - 'GStreamer-AV1-V4L2SL-Gst1.0'
```

Also, a gst+fluster rootfs for testing is now available here: https://storage.staging.kernelci.org/rootfs/bookworm-gst-fluster/20240430.0/ , so we can replace the nfsrootfs URL with the proper one.